packages/common/src/jsonApiClient/modify/update/index.js

```diff
@@ -10,7 +10,13 @@ const dep = new Dependor({
 const defaultLog = createLog('common:jsonApiClient:update')
 
 function update(
-  { openApiClient, item, log = defaultLog, resourcesToModify } = {}
+  {
+    openApiClient,
+    item,
+    log = defaultLog,
+    resourcesToModify,
```

Comment:
is `resourcesToModify` (here and in modify/create) ever something else than the item type? is it still needed after the refactor?

Reply:
I removed the check its always the item type